### DISCUSSION TITLE
--dualdevices ETH+TONDUAL

### DISCUSSION THREAD

**Discussion by feculay** [Lolliedieb/lolMiner-releases#1443] [Q&A]
HOLA, ver 1.47, QUISIERA MINAR CON TODAS MIS TARJETA ETHEREUM Y CON ALGUNAS TODUAL, PERO EL COMANDO --dualdevices NO FUNCIONA, tengo gpu 0,1,2,3,4,5,6,7 y en dual solo quiero usar las 0,1,2,3,6,7. como es la configuracion en hiveos?. gracias

ENGLISH:
HELLO, ver 1.47, I WOULD LIKE TO MINE WITH ALL MY ETHEREUM CARDS AND WITH SOME TODUAL, BUT THE --dualdevices COMMAND DOES NOT WORK, I have gpu 0,1,2,3,4,5,6,7 and in dual I only want to use the 0,1,2,3,6,7. How is the configuration in hiveos? Thank you

**Comment by mushroomjohn69** [comment]
try this instead
 

 
![git](https://user-images.githubusercontent.com/102968202/161546915-b694eabb-1a4e-4912-b8b4-063a70c9e23b.png)

**Reply by feculay** [reply]
TY
